src/main/java/org/ecommerce/util/JsonParser.java

```diff
@@ -12,4 +14,8 @@ public class JsonParser {
     public static Order parseOrder(String json) throws JsonProcessingException {
         return MAPPER.readValue(json, Order.class);
     }
+
+    public static Manager parseManager(String json) throws JsonProcessingException {
```

Comment:
Code is ok although if the number of objects to be parsed increases, please refactor JsonParser class to use generics

Reply:
You're right, here's the commit attending the recomendation https://github.com/algutierrez-griddynamics/e-commerce/pull/41/commits/3a4b6ab334f178660b15e54ac19621805c39fe76